app/posts/apply-for-teacher-training/2022-05-13-degree-autocomplete-changes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Degree autocomplete changes
+description: Why we made changes to the autocomplete
+date: 2022-05-13
+---
+
+![TODO](degree-autocomplete-changes.png "Old and new autocompletes")
+
+The section of our application where candidates enter their degree information had ‘autocomplete’ fields for degree type (BA, BSc, etc), university name and degree subject.
+
+These fields allowed users to either pick from one of the options after typing in 2 or more characters, or enter their own answer.
+
+The options provided were taken from the [HESA ITT data collection guidelines](https://www.hesa.ac.uk/collection/c21053/index).
+
+From looking at the data, we noted that a high proportion of candidates were not selecting options from the autocomplete.
+
+This led to messy data which made analysis harder, for example by having lots of different spellings and abbreviations for the same universities or subjects.
+
+We were also concerned that for the subject field, candidate may have not always realised that they could type in a subject not in the list, and may have picked the closest match instead. This may be frustrating, or cause minor issues later in the process when they have to produce evidence of their degree qualification.
+
+## What we changed
+
+To make it clearer that candidates could either select an option or type their own answer, we changed the design of the autocomplete so that the:
+
+* options have a ’Suggestions’ header
+* suggestions appear slightly below the text input, so that it looks less like a dropdown element
+* hint text explains how to use the component
```

Comment:
you show a custom message `no suggestions found, type your own answer` when there are no results?

Reply:
We don't do this - as I designed it in the prototype, if there's no matches the suggestions box disappears altogether and it just looks like a regular text input. That seemed clearer than telling people they could type their own answer (which they were already doing):

<img width="742" alt="Screenshot 2022-05-13 at 10 59 41" src="https://user-images.githubusercontent.com/30665/168260568-35b72e8b-b85d-4bbc-a11f-fdd15b36758b.png">

Looks like we've not quite done this in production yet though, opps:

<img width="725" alt="Screenshot 2022-05-13 at 11 00 01" src="https://user-images.githubusercontent.com/30665/168260690-29ea269e-140d-41ac-a3e5-efcad898f75f.png">

(I'll write up a bug card)